### ISSUE TITLE
Cancel file upload

### ISSUE THREAD

**Issue by rugk** [tootsuite/mastodon#7713]
When you upload a file (video or so) you cannot camel the upload. But while it is loading, you may have already decided to use a different file/not to attach it.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by Grazouillis** [comment]
Yes it worked before, but it's not the case anymore. It is annoying...

**Comment by mjankowski** [comment]
To clarify, is this asking for something different than a) whatever native browser ability to stop attaching/uploading exists, b) the ability to remove an uploaded/attached image from a status in compose area before you publish?

**Comment by rugk** [comment]
Given my phrasing "while it is loading" 8 years ago… I guess it's about the first aka really cancelling the upload. IIRC removing an uploaded file is already possible, but if you upload a large video on a bad network or so, you may actually experience this UX problem. 🙃 (just guessing here, I no longer mean what triggered me to feature request this one 8 years ago 🙃)
